public/index.php

```diff
@@ -7,6 +7,9 @@
 (function () {
     require __DIR__ . '/../vendor/autoload.php';
 
+    /* loading .env variables */
+    (new \Dotenv\Dotenv(__DIR__ . '/..'))->load();
```

Comment:
I didn't get it... Are you saying to load the .env variables via a middleware and pipe it to run every request?